### ISSUE TITLE
Docker explorer: all icons look the same...

### ISSUE THREAD

**Issue by weinand** [Microsoft/vscode-docker#117]
testing https://github.com/Microsoft/vscode/issues/33053:

- run something in the containers so that there is a running container and a terminated container showing in the Docker explorer

Observe:
- Images and Container should have different icons (if they need icons at all)
- the icon of the running container looks almost identical to the icon of the terminated containers

![2017-08-30_15-34-14](https://user-images.githubusercontent.com/1898161/29875115-2ee4cc6a-8d99-11e7-995b-280577b59fb9.gif)

The visual difference between the states should be bigger.

**Comment by shizn** [comment]
Just curious, when will the explorer will be released? Looks like it's not in v0.0.16.

**Comment by chrisdias** [comment]
Sorry for the delay... i hope to have it published by end of this week, if not then early next week.

In the meantime, you can always clone and build locally.

``` bash
git clone https://github.com/microsoft/vscode-docker
cd vscode-docker
npm install
npm install -g vsce
vsce package
code-insiders --install-extension vscode-docker-0.0.16.vsix
```
